project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.7.2
+sbt.version=1.9.7
```

Comment:
sbt can go higher 1.9.9

Reply:
@derickdamoah I tend to align to latest versions shown within tax catalogue even if not latest version.